mypy.ini

```diff
@@ -1,2 +1,14 @@
 [mypy]
 ignore_missing_imports = True
+
+[mypy-gpflow.conditionals.*,gpflow.config.*,gpflow.covariances.*,gpflow.expectations.*]
+ignore_errors = True
+
+[mypy-gpflow.kernels.*,gpflow.models.*,gpflow.monitor.*,gpflow.optimizers.*,gpflow.utilities.*]
+ignore_errors = True
+
+[mypy-gpflow.mean_functions]
+ignore_errors = True
+
+[mypy-tests.*]
+ignore_errors = True
```

Comment:
these are the modules that haven't been type-fixed yet ?

Reply:
exactly